src/web/components/ClientSideTokenGeneration/CstgDomain.tsx

```diff
@@ -57,7 +74,41 @@ export function CstgDomainItem({ domain, onClick, onDelete, checked }: CstgDomai
       <td className='domain'>{domain}</td>
       <td className='action'>
         <div className='action-cell'>
-          <DeleteConfirmationDialog domain={domain} onRemoveDomain={onDelete} />
+          <button
+            type='button'
+            className='icon-button'
+            title='Edit'
+            onClick={() => {
+              setShowEditDialog(true);
+            }}
+          >
+            <FontAwesomeIcon icon='pencil' />
+          </button>
```

Comment:
Probably would create another ticket for this but thoughts about having EditButton component and DeleteButton component?  We are doing this in several places now with editing domains, key pairs, participant stuff and API keys.

Reply:
Sounds like a great idea!